URL-API.md

```diff
@@ -1,80 +1,293 @@
+* `start` defines the geographical point in a comma-separated list.
+* `desttype` In the example, a POI is used as destination type.
+* `dest` In the example the POI ID for MazeMap office is used as destination.
+
+
+## Specifying custom names for points
+
+When starting MazeMap either with predefined points (e.g. if you have defined a path), you can input a custom string for the names of these points. I.e. if you you are linking to an office with id 404, you can specify that it should display as "John's Office".
+
+```
+http://use.mazemap.com/?campusid=1&desttype=point&dest=10.4035968,63.4175039,6&starttype=point&start=10.4030281,63.4185463,0&startname=Start Here&destname=John's Office
+```
+
+* `startname` defines the string to be used for the start point.
+* `destname` defines the string to be used for the destination point.
+
+If you defined a `sharepoi`, use `sharepoiname` to customize the name.
+
+```
+http://use.mazemap.com/?campusid=1&sharepoitype=point&sharepoi=10.40153,63.41809,1&sharepoiname=Vending Machine X
+```
+
+
+## Specifying a custom zoom
+
+To override the default zoom:
+
+```
+http://use.mazemap.com/?campusid=1&zoom=17
+```
+
+* `zoom` can have a value between 1-22
+
+
+## Specifying a custom view
+
+To override the default center point of the view, use the `viewtype` and `view` parameters. The syntax was discussed in general under 'POIs in general'. The value of `viewtype` defines the format expected by the `view` parameter.
+
+```
+viewtype=poi&view=<MazeMap POI ID>
+viewtype=point&view=<longitude>,<latitude>,<floor>
+viewtype=identifier&view=<Customer defined ID>
+```
+
+To center the starting point of the path defined above (instead of centering the path which is default):
+
+```
+http://use.mazemap.com/?campusid=1&viewtype=point&view=10.40290,63.41860,0&starttype=point&start=10.40290,63.41860,0&desttype=identifier&dest=322-620
+```
+
+* `viewtype` defines the format expected by the `view` parameter. In this case, a `point`.
+* `view` is set to be the same point as `start`
+
+
+## Simulating a position
+
+To simulate the position of the user, use the `postype` and `pos` parameters. The syntax was discussed in general under 'POIs in general'. The value of `postype` defines the format expected by the `pos` parameter.
+
+```
+postype=poi&pos=<MazeMap POI ID>
+postype=point&pos=<longitude>,<latitude>,<floor>
+postype=identifier&pos=<Customer defined ID>
+```
+
+```
+http://use.mazemap.com/?campusid=1&postype=point&pos=10.40213,63.41879,0
+```
+
+* `postype` defines the format expected by the `pos` parameter. In this case, a `point`.
+* `pos` defines the geographical point in a comma-separated list.
+
+#### Warning!
+
+If not used in the correct way, simulating positions can easily confuse the user. Be careful using it for other purposes than stationary info screens.
+
+
+## Specifying the placement of 'Open in new window' link
```

Comment:
I would put these things under a seperate "EMBED" text file. It belongs together general info about embedding the maps in iframes, etc.